script.js

```diff
@@ -0,0 +1,306 @@
+const menubar = document.querySelector('.hamburger');
+const navmenu = document.querySelector('.nav-menu');
+menubar.addEventListener('click', () => {
+  menubar.classList.toggle('active');
+  navmenu.classList.toggle('active');
+});
+
+// Popup window section
+
+const array = [
+  {
+    name: 'Keeping track of hundreds of components',
+    desc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.   Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    mobileImg: './images/pop-up/mobile.png',
+    liveImg: './images/pop-up/button-icons/Icon.png',
+    githubImg: './images/pop-up/button-icons/github.png',
+    desktopImg: './images/pop-up/Snapshoot Portfolio.png',
+    tech: [
+      'Ruby on rails',
+      'css',
+      'JavaScript',
+      'Bootstrap',
+      'Terminal',
+      'Codepen',
+    ],
+  },
+];
+
+const body = document.querySelector('body');
+const openerButton = document.querySelector('.button1');
+
+openerButton.addEventListener('click', () => {
+  const main = document.createElement('div');
+  main.className = 'main';
+  const maincard = document.createElement('div');
+  maincard.className = 'maincard';
+  maincard.innerHTML = `
+
+  <button class="close-button2">&times;</button>
+  <div>
+ 
+  <img  class = "popup-image"src="${array[0].mobileImg}" alt="">
+  <img  class = "popup-image2"src="${array[0].desktopImg}" alt="">
+ </div>
+ <div class="headline">
+    <h2 class="popup-par">${array[0].name}</h2>
+    <div class ="frame18">
+       <button class="first-button2" type="button" id="messagep">See live <img src="${array[0].liveImg}" alt=""></button>
+       <button class="first-button1" type="button" id="messagep">See source <img src="${array[0].githubImg}" alt=""></button>
+    </div>
+  </div>     
+    <ul class="popup-list">
+      <li class="ruby">${array[0].tech[0]}</li>
```

Comment:
[OPTIONAL] Good job done by implementing this part, but I would suggest that you assign ids with the same id in your array items to each of these buttons and create a function that displays the popup and make receive the id as argument this will let you call the dispalyPopup in a for loop for each of your buttons, avoid repeating the same thing and make your code DRY